javascript-challenges.test.js

```diff
@@ -14,30 +16,121 @@
 
 // a) Create a test with expect statements using the variables provided.
 
-const secretCodeWord1 = "Lackadaisical"
-// Expected output: "L4ck4d41s1c4l"
-const secretCodeWord2 = "Gobbledygook"
-// Expected output: "G0bbl3dyg00k"
-const secretCodeWord3 = "Eccentric"
-// Expected output: "3cc3ntr1c"
+
+describe('Secret Code Message', () => {
+    it('encodes "Lackadaisical" to "L4ck4d41s1c4l"', () => {
```

Comment:
Your tests look good, I think this could be condensed down to one it block if you wanted to simplify things.  The intent of the function should remain the same and that is what you are testing with different test cases.